sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkWiringIntegrationTest.java

```diff
@@ -74,4 +74,33 @@ public void verifyStreamActions() {
 
     Assertions.assertEquals(3, messageList.size());
   }
+
+  @Test
+  public void verifyCounterEventSourcedWiring() {
+
+    String counterIncrease =
+        webClient
+            .post()
+            .uri("/counter/hello/increase/10")
+            .retrieve()
+            .bodyToMono(String.class)
+            .block(timeout);
+
+    Assertions.assertEquals("\"10\"", counterIncrease);
+
+    String counterMultiply =
+        webClient
+            .post()
+            .uri("/counter/hello/multiply/20")
+            .retrieve()
+            .bodyToMono(String.class)
+            .block(timeout);
+
+    Assertions.assertEquals("\"200\"", counterMultiply);
+
+    String counterGet =
+        webClient.get().uri("/counter/hello").retrieve().bodyToMono(String.class).block(timeout);
```

Comment:
How can a user know where the id of the entity is supposed to be passed? Like here `hello`

Reply:
It's defined on the `RequestMapping` of the Entity but I'm pretty sure you know that.. :D so I'm not sure I understood the question.. Could you elaborate please? 🙇

Reply:
:)) No sir. I know little about Spring and totally missed it so I bow back to you :)

Reply:
oh fair.. yeah, it definitely feels weird if you haven't seen it before.. cool :)